src/EntraModuleBuilder.ps1

```diff
@@ -443,7 +468,17 @@ foreach (`$subModule in `$subModules) {
         Log-Message "[EntraModuleBuilder]: Creating manifest for $moduleName at $manifestPath"
         try{
              New-ModuleManifest @moduleSettings
-        Update-ModuleManifest -Path $manifestPath -PrivateData $PSData
+             Update-ModuleManifest -Path $manifestPath -PrivateData $PSData
```

Comment:
The parameters in `Update-ModuleManifest` are also in `New-ModuleManifest`.
Why can't we run one command `New-ModuleManifest`?

Reply:
When you run the New-ModuleManifest you may pass all the necessary parameters one at a time e.g. Path, RootModule, GUID etc as one or use an object (HashTable) above. Any fields not set will be added but commented out in the output manifest file. 

However, when you do that, you find that the PrivateData is not properly nested.  It should be PrivateData=@{ @{PSData=$PSData}}. But when you pass it directly in the New-ModuleManifest, the nesting is wrong. So, you create the manifest and then use Update-ModuleManifest  to update the PrivateData section and this guarantees proper nesting.